dev/com.ibm.ws.microprofile.openapi.ui/resources/com/ibm/ws/microprofile/openapi/ui/resources/OpenAPIUIMessages.nlsprops

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR Open API UI
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_NONE
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# Message prefix block: CWWKO1750 - CWWKO1799
+
+CWWKO1750W_OPEN_API_UI_PATH_RESERVED=CWWKO1750W: The OpenAPI UI Path cannot be set to {0}. This Path is reserved for use by the application server.
+CWWKO1750W_OPEN_API_UI_PATH_RESERVED.explanation=The OpenAPI UI Path cannot be set to a value that has been reserved for use by the application server.
```

Comment:
```suggestion
CWWKO1750W_OPEN_API_UI_PATH_RESERVED.explanation= The application server reserves certain values for use, preventing the OpenAPI UI Path from being set to these values.
```

Reply:
This message is used solely for one other endpoint - so have focused the message on that conflict instead of a generic message for any potential endpoint value